app/routes.py

```diff
@@ -1 +1,209 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, request, make_response, jsonify, abort
+from app.models.task import Task
+from app.models.goal import Goal
+from app import db
+from datetime import datetime
+import requests, os
+
+task_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+
+@task_bp.route("", methods=["POST"])
+def create_tasks():
+    request_body = request.get_json()
+
+    try:
+        new_task = Task(title=request_body["title"], description=request_body["description"])
+    except KeyError:
+        return jsonify({"details": "Invalid data"}), 400
```

Comment:
Good use of try-except, that I would want to tell the user what data is invalid and make a more meaningful error message.